manager/src/main/java/com/redhat/service/smartevents/manager/dao/BridgeDAO.java

```diff
@@ -34,10 +38,22 @@ public Bridge findByIdAndCustomerId(String id, String customerId) {
         return find("#BRIDGE.findByIdAndCustomerId", params).firstResult();
     }
 
-    public ListResult<Bridge> findByCustomerId(String customerId, QueryInfo queryInfo) {
+    public ListResult<Bridge> findByCustomerId(String customerId, QueryResourceInfo queryInfo) {
         Parameters parameters = Parameters.with("customerId", customerId);
-        long total = find("#BRIDGE.findByCustomerId", parameters).count();
-        List<Bridge> bridges = find("#BRIDGE.findByCustomerId", parameters).page(queryInfo.getPageNumber(), queryInfo.getPageSize()).list();
+        PanacheQuery<Bridge> query = find("#BRIDGE.findByCustomerId", parameters);
+
+        String filterName = queryInfo.getFilterInfo().getFilterName();
+        Set<ManagedResourceStatus> filterStatus = queryInfo.getFilterInfo().getFilterStatus();
+        if (Objects.nonNull(filterName)) {
+            query.filter("byName", Parameters.with("name", filterName + "%"));
+        }
+        if (Objects.nonNull(filterStatus) && !filterStatus.isEmpty()) {
+            query.filter("byStatus", Parameters.with("status", filterStatus));
```

Comment:
Without the hackery magic (see below) I'd need to convert the statuses to `String`.